go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	github.com/aws/aws-sdk-go v1.41.19
 	github.com/imdario/mergo v0.3.12 // indirect
-	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
```

Comment:
This entry caused the following problem for me:
```
$ make
go fmt ./...
go: github.com/openshift/api@v3.9.1-0.20190924102528-32369d4db2ad+incompatible: invalid pseudo-version: preceding tag (v3.9.0) not found
make: *** [Makefile:3: build] Error 1
```

Might be an issue on my side, was this version pinned for a specific reason?